srcpkgs/clex/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'clex'
+pkgname=clex
+version=4.6.patch9
+revision=1
+archs="x86_64 i868 x86_64-musl"
```

Comment:
Why is it restricted to these architectures? If there is no good reason, remove the line.

Reply:
I restricted it as it fails to pass the checks on arm based arches, due to 
> `CLEX requires a CURSES library with a wide character support`

I am currently unsure which dependency I need to add to fix said error, I'll work on fixing that.